pysatSpaceWeather/instruments/sw_f107.py

```diff
@@ -9,12 +9,22 @@
     'f107'
 tag
     - '' LASP F10.7 data (downloads by month, loads by day)
-    - 'all' All LASP standard F10.7
+    - 'all' All LASP standard F10.7 (deprecated)
     - 'prelim' Preliminary SWPC daily solar indices
     - 'daily' Daily SWPC solar indices (contains last 30 days)
     - 'forecast' Grab forecast data from SWPC (next 3 days)
     - '45day' 45-Day Forecast data from the Air Force
 
+Example
+-------
+Download and load all of the historic F10.7 data
+::
+
+    f107 = pysat.Instrument('sw', 'f107')
+    f107.download(start=f107.lasp_stime, stop=f107.today(), freq='MS')
+    f107.load(date=f107.lasp_stime, end_date=f107.today())
```

Comment:
`download` has inclusive dates while `load` has an exclusive last date.
```suggestion
    f107.load(date=f107.lasp_stime, end_date=f107.tomorrow())
```

Reply:
Similarly for F10.7 etc.

Reply:
This would be worse, since the data doesn't come close to 'today', and would result in an additional empty load attempt.

Reply:
Fair enough, the comment was based on the presumption there would be recent data and a user may wonder why they didn't get the full data range.